starboard/evergreen/testing/run_all_tests.sh

```diff
@@ -24,7 +24,7 @@ AUTH_METHOD="public-key"
 USE_COMPRESSED_SYSTEM_IMAGE="false"
 SYSTEM_IMAGE_EXTENSION=".so"
 
-DISABLE_TESTS="false"
+DISABLE_TESTS="true"
```

Comment:
Thanks for doing this. Can you please just add a comment linking to the bug? Something like

`# TODO(b/232134303): enable once updated trunk binaries are released.`